backend/coins/btc/account_test.go

```diff
@@ -190,11 +189,26 @@ func TestSignAddress(t *testing.T) {
 	account := mockAccount(t, nil)
 	require.NoError(t, account.Initialize())
 	// pt2r is not an available script type in the mocked account.
-	_, _, err := btc.SignBTCAddress(account, "Hello there", signing.ScriptTypeP2TR)
+	_, _, err := SignBTCAddress(account, "Hello there", signing.ScriptTypeP2TR)
 	require.Error(t, err)
-	address, signature, err := btc.SignBTCAddress(account, "Hello there", signing.ScriptTypeP2WPKH)
+	address, signature, err := SignBTCAddress(account, "Hello there", signing.ScriptTypeP2WPKH)
 	require.NoError(t, err)
 	require.NotEmpty(t, address)
 	require.Equal(t, base64.StdEncoding.EncodeToString([]byte("signature")), signature)
 
 }
+
+func TestIsChange(t *testing.T) {
```

Comment:
Would be nice also to test for false positives: non-change addresses should not be mistakenly identified as change